packages/form-js-editor/src/render/components/properties-panel/entries/DefaultValueEntry.js

```diff
@@ -11,10 +11,10 @@ import { get } from 'min-dash';
 
 import { useService } from '../../../hooks';
 
-import { INPUTS } from '../Util';
+import { INPUTS, OPTIONS_INPUTS } from '../Util';
 
 
-export default function DefaultValueEntry(props) {
+export default function DefaultOptionEntry(props) {
```

Comment:
Did we make the conscious decision to rename `values` to options? What is the benefit?